jsk_perception/CMakeLists.txt

```diff
@@ -246,23 +246,25 @@ add_dependencies(hough_lines              ${PROJECT_NAME}_gencfg ${PROJECT_NAME}
 add_dependencies(rectangle_detector       ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_gencpp)
 add_dependencies(color_histogram_sliding_matcher       ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_gencpp)
 
+# FIXME: actually, roseus executable is usable only on Linux
+if(GNU)
```

Comment:
because roseus can be compiled but it segfaults at ros::roseus.

--
和田 健太郎 / Kentaro Wada
http://wkentaro.com/

2016年4月26日 9:06 +0900 に Kei Okada<notifications@github.com>は書きました:
>  
> Injsk_perception/CMakeLists.txt(https://github.com/jsk-ros-pkg/jsk_recognition/pull/1605#discussion_r61011324):
>  
> >@@ -246,23 +246,25 @@ add_dependencies(hough_lines       ${PROJECT_NAME}_gencfg ${PROJECT_NAME}>add_dependencies(rectangle_detector    ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_gencpp)>add_dependencies(color_histogram_sliding_matcher    ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_gencpp)>>+# FIXME: actually, roseus executable is usable only on Linux>+if(GNU)
>  
> why not using roseus_FOUND or find_program
>  
>  
> —
> You are receiving this because you authored the thread.
> Reply to this email directly orview it on GitHub(https://github.com/jsk-ros-pkg/jsk_recognition/pull/1605/files/18164dc33da667b13fd63c7de564444b416968b6#r61011324)
>  
>  
>